src/modules/Auth/AuthCallback.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Redirect } from 'react-router-dom';
+
+// const extractParams = (hash) => {
```

Comment:
Remove this commented code if not needed

Reply:
Done.